autoexec.cfg

```diff
@@ -59,7 +59,7 @@ fullbright_off
 // Sensitivity toggle
 sar_function sensitivity_reset   "sensitivity $sensitivity"
 sar_function sensitivity_capture "svar_from_cvar sensitivity sensitivity"
-sar_alias sensitivity_on  "sar_alias sensitivity_toggle sensitivity_off; sensitivity_capture; sensitivity 0.1"
+sar_alias sensitivity_on  "sar_alias sensitivity_toggle sensitivity_off; sensitivity_capture; sensitivity .1"
```

Comment:
I would do this consistency change the other way round; i.e. adding a leading 0 to all the other numbers